exporters/otlp/src/otlp_grpc_exporter_factory.cc

```diff
@@ -0,0 +1,32 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// Make sure to include GRPC exporter first because otherwise Abseil may create
+// ambiguity with `nostd::variant`. See issue:
+// https://github.com/open-telemetry/opentelemetry-cpp/issues/880
+#include "opentelemetry/exporters/otlp/otlp_grpc_exporter.h"
+
+#include "opentelemetry/exporters/otlp/otlp_grpc_exporter_factory.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace otlp
+{
+
+std::unique_ptr<opentelemetry::sdk::trace::SpanExporter> OtlpGrpcExporterFactory::Create()
+{
+  OtlpGrpcExporterOptions options;
+  return Create(options);
```

Comment:
Should we use `Create(OtlpGrpcExporterOptions{})` to reduce copy here?

Reply:
Thanks for the comments.
Here the Create() method with an option takes a `const OtlpGrpcExporterOptions &options` parameter,
so there should be no copy.